include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -1157,4 +1157,12 @@ def ZOp : OneTargetOp<"z", [Hermitian]> {
   }];
 }
 
+def CustomUnitarySymbolOp : QuakeOperator<"unitary"> {
```

Comment:
Will this replace the Op in the other PR then?

Reply:
Yes, that is the intention. I will add a comment there.